apps/src/templates/sectionAssessments/FreeResponsesAssessments.jsx

```diff
@@ -23,7 +46,7 @@ class FreeResponsesAssessments extends Component {
   };
 
   state = {
-    [COLUMNS.NAME]: {
+    [COLUMNS.STUDENT]: {
```

Comment:
would it be better to change the constant to use `NAME`? so that it's consistent with the other column naming?

Reply:
Good point!